# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1455694049 28800
#      Tue Feb 16 23:27:29 2016 -0800
# Node ID 7abdce2ae9000170b02e04da2e0a7ca872372981
# Parent  b76955e83fe4ff45cf973b71132b10ef77930dd8
maint: Add missing silent rules for generating make files

* doc/module.mk, libinterp/module.mk: Prefix more rules with $(AM_V_GEN).

diff --git a/doc/module.mk b/doc/module.mk
--- a/doc/module.mk
+++ b/doc/module.mk
@@ -547,17 +547,17 @@ doc/refcard/refcard-legal.ps: doc/refcar
 doc/refcard/refcard-letter.pdf: doc/refcard/refcard.tex
 doc/refcard/refcard-letter.dvi: doc/refcard/refcard.tex
 doc/refcard/refcard-letter.ps: doc/refcard/refcard-letter.dvi
 	-$(AM_V_DVIPS)$(DVIPS) $(AM_V_texinfo) -T 11in,8.5in -o $@ $<
 
 DIRSTAMP_FILES += doc/refcard/$(octave_dirstamp)
 
 $(srcdir)/doc/interpreter/images.mk: $(srcdir)/doc/interpreter/config-images.sh $(srcdir)/doc/interpreter/images.awk $(srcdir)/doc/interpreter/images
-	$(SHELL) $(srcdir)/doc/interpreter/config-images.sh $(top_srcdir)
+	$(AM_V_GEN)$(SHELL) $(srcdir)/doc/interpreter/config-images.sh $(top_srcdir)
 
 $(refcard_DVI) : %.dvi : %.tex | doc/refcard/$(octave_dirstamp)
 	-$(AM_V_TEX)cd $(@D) && \
 	TEXINPUTS="$(abs_top_srcdir)/doc/refcard:$(TEXINPUTS):" \
 	$(TEX) $(<F) $(AM_V_texidevnull)
 
 $(refcard_PDF) : %.pdf : %.tex | doc/refcard/$(octave_dirstamp)
 	-$(AM_V_PDFTEX)cd $(@D) && \
diff --git a/libinterp/module.mk b/libinterp/module.mk
--- a/libinterp/module.mk
+++ b/libinterp/module.mk
@@ -131,17 +131,17 @@ DIST_SRC += \
 include libinterp/parse-tree/module.mk
 include libinterp/octave-value/module.mk
 include libinterp/operators/module.mk
 include libinterp/template-inst/module.mk
 include libinterp/corefcn/module.mk
 include libinterp/dldfcn/module.mk
 
 $(srcdir)/libinterp/dldfcn/module.mk: $(srcdir)/libinterp/dldfcn/config-module.sh $(srcdir)/libinterp/dldfcn/config-module.awk $(srcdir)/libinterp/dldfcn/module-files
-	$(SHELL) $(srcdir)/libinterp/dldfcn/config-module.sh $(top_srcdir)
+	$(AM_V_GEN)$(SHELL) $(srcdir)/libinterp/dldfcn/config-module.sh $(top_srcdir)
 
 if AMCOND_ENABLE_DYNAMIC_LINKING
   OCT_FILES = $(DLDFCN_LIBS:.la=.oct)
   DLD_LIBOCTINTERP_LIBADD = libinterp/liboctinterp.la
 else
   OCT_FILES =
   DLD_LIBOCTINTERP_LIBADD =
 endif
